components/Task.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styled from "styled-components";
+import TaskItem from "./TaskItem";
+
+const initialTasks = [
+  {
+    id: Math.random().toString(36).substring(2),
+    taskName: "Reading",
+    duration: "30min",
+  },
+  {
+    id: Math.random().toString(36).substring(2),
+    taskName: "Capstone Project",
+    duration: "6h",
+  },
+  {
+    id: Math.random().toString(36).substring(2),
```

Comment:
Nice!

Reply:
You could write this as a function `generateID`, that would be more DRY and readable.